doc/develop/fixing_data.rst

```diff
@@ -354,6 +365,40 @@ For example for monthly data, place the files in the ``/Tier3/MSWEP/latestversio
 
 For more info: http://www.gloh2o.org/
 
+.. _fixing_native_models:
+
+Native models : IPSL-CM6,...
+-----------------------------
+
+The following models are natively supported through the procedure
+described above (:ref:`fix_structure`) and at
+:ref:`configure_native_models`:
+
+  - **IPSL-CM6** : both output formats (i.e. the ``Output`` and the
+    ``Analyse / Time series`` formats) are supported, and should be
+    configured in recipes as e.g.:
+
+    .. code-block:: yaml
+
+      datasets:
+        - {simulation: CM61-LR-hist-03.1950, exp: piControl, freq: Analyse/TS_MO,
+           account: p86caub,  status: PROD, dataset: IPSL-CM6, project: IPSLCM,
+           root: /thredds/tgcc/store}
+        - {simulation: CM61-LR-hist-03.1950, exp: historical, freq: Output/MO,
+           account: p86caub,  status: PROD, dataset: IPSL-CM6, project: IPSLCM,
+           root: /thredds/tgcc/store}
+
+    .. _ipslcm_extra_facets_example:
```

Comment:
Is this intended as a label for the yaml block? In that case I think it has to go before the block, i.e.
```
.. _ipslcm_extra_facets_example:

.. code-block:: yaml
```

Reply:
No, it is a ref for the text below